src/components/Navigator/Navigator.tsx

```diff
@@ -40,83 +33,89 @@ const NavigatorTab: React.FC<NavigatorTabProps> = ({ label }) => {
   return <NavigatorTabWrapper>{label}</NavigatorTabWrapper>;
 };
 
-const NavigatorBodyInner: React.FC<NavigatorBodyInnerProps> = ({
-  items,
-  currentTime,
-}) => {
-  return items.map((cue) => {
-    return (
-      <NavigatorItem
-        text={cue.text}
-        startTime={cue.startTime}
-        t={currentTime}
-        key={cue.id}
-      />
-    );
-  });
+interface Cue {
+  id: number;
+  startTime: string;
+  endTime: string;
+  text: string;
+}
+
+interface NavigatorBodyInnerProps {
+  currentTime: number;
+  items: Cue[];
+}
+
+const NavigatorBodyInner: React.FC<NavigatorBodyInnerProps> = ({ items, currentTime }) => {
+  return (
+    <>
+      {items.map(({ id, text, startTime }) => (
+        <NavigatorItem label={text} startTime={startTime} t={currentTime} key={id} />
+      ))}
+    </>
+  );
 };
 
-const NavigatorWrapper = styled("div", {
-  display: "flex",
-  flexDirection: "column",
-  width: "100%",
-  height: "100%",
-  flexGrow: "1",
-  flexShrink: "0",
-  position: "relative",
-  zIndex: "1",
-  boxShadow: "-5px 0 5px #00000011",
+const NavigatorWrapper = styled('div', {
+  display: 'flex',
+  flexDirection: 'column',
+  width: '100%',
+  height: '100%',
+  flexGrow: '1',
+  flexShrink: '0',
+  position: 'relative',
+  zIndex: '1',
+  boxShadow: '-5px 0 5px #00000011',
```

Comment:
Just curious @adamjarling, is this something linter was yelling about?

Reply:
If you mean the single quotes, no.  Guess that was an oversight to include that in the prettier config file.  I'm either way on quotes.... and actually am used to Prettier's default way of double quotes.  We can switch it back if ya like.